File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Adma/ADM101UGrdGroupItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Adma
{
    [Serializable]
	public class ADM101UGrdGroupItemInfo
	{

		private String _grpId;
		private String _grpNm;
		private String _grpSeq;
		private String _grpDesc;
		private String _dispGrpId;
		private String _dataRowState;

		public String GrpId
		{
			get { return this._grpId; }
			set { this._grpId = value; }
		}

		public String GrpNm
		{
			get { return this._grpNm; }
			set { this._grpNm = value; }
		}

		public String GrpSeq
		{
			get { return this._grpSeq; }
			set { this._grpSeq = value; }
		}

		public String GrpDesc
		{
			get { return this._grpDesc; }
			set { this._grpDesc = value; }
		}

		public String DispGrpId
		{
			get { return this._dispGrpId; }
			set { this._dispGrpId = value; }
		}

		public String DataRowState
		{
			get { return this._dataRowState; }
			set { this._dataRowState = value; }
		}

		public ADM101UGrdGroupItemInfo() { }

		public ADM101UGrdGroupItemInfo(String grpId, String grpNm, String grpSeq, String grpDesc, String dispGrpId, String dataRowState)
		{
			this._grpId = grpId;
			this._grpNm = grpNm;
			this._grpSeq = grpSeq;
			this._grpDesc = grpDesc;
			this._dispGrpId = dispGrpId;
			this._dataRowState = dataRowState;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/adma/ADM101UGrdGroupItemInfo.java
package nta.med.data.model.ihis.adma;

public class ADM101UGrdGroupItemInfo {
	private String grpId      ;
	private String grpNm      ;
	private Double grpSeq     ;
	private String grpDesc    ;
	private String dispGrpId ;
	public ADM101UGrdGroupItemInfo(String grpId, String grpNm, Double grpSeq,
			String grpDesc, String dispGrpId) {
		super();
		this.grpId = grpId;
		this.grpNm = grpNm;
		this.grpSeq = grpSeq;
		this.grpDesc = grpDesc;
		this.dispGrpId = dispGrpId;
	}
	public String getGrpId() {
		return grpId;
	}
	public void setGrpId(String grpId) {
		this.grpId = grpId;
	}
	public String getGrpNm() {
		return grpNm;
	}
	public void setGrpNm(String grpNm) {
		this.grpNm = grpNm;
	}
	public Double getGrpSeq() {
		return grpSeq;
	}
	public void setGrpSeq(Double grpSeq) {
		this.grpSeq = grpSeq;
	}
	public String getGrpDesc() {
		return grpDesc;
	}
	public void setGrpDesc(String grpDesc) {
		this.grpDesc = grpDesc;
	}
	public String getDispGrpId() {
		return dispGrpId;
	}
	public void setDispGrpId(String dispGrpId) {
		this.dispGrpId = dispGrpId;
	}
}
